reference/employer/openapi.yaml

```diff
@@ -20,6 +20,8 @@ info:
     - `payment`: Read payroll and contractor related payments by the company
 
     - `pay_statement`: Read detailed pay statements for each individual
+
+    - `deduction`: Create and manage benefits and enrollment within a company
```

Comment:
Side note: I wonder if we should rename the scope? I left it the same during implementation only because it'd require migration but perhaps worth asserting one way or another.

Reply:
yeah karthik mentioned this in the other PR. maybe we could add in some backward compatibility